vcd/datasource_vcd_independent_disk_test.go

```diff
@@ -73,6 +80,7 @@ func TestAccVcdDataSourceIndependentDisk(t *testing.T) {
 					resource.TestCheckResourceAttr("data.vcd_independent_disk."+datasourceName, "bus_type", "SCSI"),
 					resource.TestCheckResourceAttr("data.vcd_independent_disk."+datasourceName, "bus_sub_type", "lsilogicsas"),
 					resource.TestCheckResourceAttr("data.vcd_independent_disk."+datasourceName, "storage_profile", "*"),
+					resource.TestCheckResourceAttr("data.vcd_independent_disk."+datasourceName, "metadata.key1", "value1"),
```

Comment:
please, move `value1` and `value2` to `params` maps with defined keys. Similar we have in other tests.

Reply:
Done, all is parameterized now!